concepts/switch-statement/about.md

```diff
@@ -145,6 +145,23 @@ Starting with Java 14 (available as a preview before in Java 12 and 13) it is po
    However if you use the new `->` notation it must be followed by either: a single statement/expression, a `throw` statement or a `{}` block.
    No more confusion!
 
+The first LTS (Long Term Support) version that had these improvements was Java 17, released on September, 2021.
```

Comment:
This sentence isn't needed here because line 34 already states this came out of beta in Java 14.

```suggestion
```